benchmark/flexsn/benchmark_vgg_inductor.py

```
@@ -110,30 +110,22 @@ def main() -> None:
-
-        # One extra iteration to initialize states and trigger Triton JIT +
-        # autotune before timing begins (avoids measuring compile cost).
-        step()
-        ms = cuda_time_ms(step)
+        model.eval()
+
+        with torch.no_grad():
+            # One extra forward to initialise states and trigger Triton JIT +
+            # autotune before timing begins (avoids measuring compile cost).
+            model(x)
+            ms = cuda_time_ms(lambda: (functional.reset_net(model), model(x)))
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `model`

(B023)